datasets/human-cell-atlas.yaml

```diff
@@ -0,0 +1,70 @@
+Name: Human Cell Atlas
+
+Description: "The Human Cell Atlas (HCA) is a collaborative community of
+  international scientists. Our mission is to create comprehensive reference
+  maps of all the cells in the human body as a basis for both understanding
+  human health and diagnosing, monitoring, and treating disease. The HCA
+  registry has more than one thousand member scientists from hundreds of
+  institutions around the world. The project is steered and governed by an
+  Organizing Committee, co-chaired by Aviv Regev and Sarah Teichmann."
+
+Documentation: https://data.humancellatlas.org/
+
+Contact: https://data.humancellatlas.org/contact
+
+ManagedBy: Genomics Institute, University of California, Santa Cruz (UCSC)
+
+UpdateFrequency: Monthly
+
+Tags:
+  - biology
+  - cell biology
+  - genome
+  - genomic
+  - transcriptomics
+  - gene expression
+  - single-cell transcriptomics
+  - cell imaging
+  - Homo sapiens
+  - Mus musculus
+
+License: https://data.humancellatlas.org/about/data-use-agreement
+
+Citation: "The URL for the HCA Data Portal, https://data.humancellatlas.org, can
+  be used as the citation for data obtained from the HCA. Alternatively, you can
+  cite: Regev A., et al. 2017. The Human Cell Atlas. Elife. Dec 5;6. pii:
+  e27041. doi: 10.7554/eLife.27041."
+
+Resources:
+  - Description: "An S3 bucket containing all publicly accessible data files in 
+      the Human Cell Atlas. The bucket layout is documented at TODO. Metadata 
+      can be viewed at https://explore.data.humancellatlas.org/ or accessed 
+      programmatically at https://service.azul.data.humancellatlas.org/"
+    ARN: arn:aws:s3:::TODO
+    Region: us-east1
+    Type: S3 bucket
+    Explore: 
+      - https://explore.data.humancellatlas.org/
+      - https://service.azul.data.humancellatlas.org/
+
+DataAtWork:
```

Comment:
This whole entry is optional. I think we should start with some Publications and omit *Tutorials* and *Tools & Applications* for the initial submission.